pyproject.toml

```
@@ -32,7 +32,7 @@ filterwarnings = [
 line-length = 100
 
 [tool.coverage.run]
-omit = ["corgi/core/migrations/*"]
+omit = ["corgi/core/migrations/*","corgi/collectors/migrations/*","corgi/tasks/management/commands/*"]
```

Not sure if below will work:
```suggestion
omit = ["corgi/**/migrations/*"]
```

I'd like to keep reporting coverage for management commands. The updatecomponenttaxonomy command is currently broken, and when I fix it I'm planning to also add a test, so coverage stats for these are still useful IMO.